### DISCUSSION TITLE
Method to pause autoplay

### DISCUSSION THREAD

**Discussion by berianwilliams** [sachinchoolur/lightGallery#1204] [Q&A]
@sachinchoolur Is there a way of stopping and starting autoplay dynamically?  I would like to pause the scrolling inside my own event handlers, as well as having the normal inbuilt pause button.

**Comment by sachinchoolur** [comment]
Hey @berianwilliams,

We have already received a few requests to implement public methods to stop and pause autoplay.
I'll include it in the next version.

**Reply by berianwilliams** [reply]
@sachinchoolur Did this end up happening, please?
